pkgs/applications/window-managers/i3/kitti3.nix

```diff
@@ -0,0 +1,45 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, fetchpatch
+, poetry-core
+, i3ipc
+, lib
+}:
+
+buildPythonApplication rec {
+  pname = "kitti3";
+  version = "unstable-2021-09-11";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "LandingEllipse";
+    repo = pname;
+    rev = "f9f94c8b9f8b61a9d085206ada470cfe755a2a92";
+    hash = "sha256-bcIzbDpIe2GKS9EcVqpjwz0IG2ixNMn06OIQpZ7PeH0=";
+  };
+
+  patches = [
+    # Fixes `build-system` not being specified in `pyproject.toml`
+    # https://github.com/LandingEllipse/kitti3/pull/25
```

Comment:
Looks like upstream is pretty dead, we could just put the patch in nixpkgs.

Reply:
Updated :+1: